cmd/eth1indexer/main.go

```diff
@@ -515,6 +515,19 @@ func HandleChainReorgs(bt *db.Bigtable, client *rpc.ErigonClient, depth int) err
 		if !bytes.Equal(nodeBlock.Hash().Bytes(), dbBlock.Hash) {
 			logrus.Warnf("found incosistency at height %v, node block hash: %x, db block hash: %x", i, nodeBlock.Hash().Bytes(), dbBlock.Hash)
 
+			// first we set the cached marker of the last block in the blocks/data table to the block prior to the forked one
+			if i > 0 {
+				previousBlock := i - 1
+				err := bt.SetLastBlockInBlocksTable(int64(previousBlock))
+				if err != nil {
+					return err
+				}
+				err = bt.SetLastBlockInDataTable(int64(previousBlock))
+				if err != nil {
+					return err
+				}
```

Comment:
Minor: The error messages could use a bit more information. Please consider something like 
```suggestion
				if err != nil {
					return fmt.Errorf("error setting last block [%v] in blocks table: %w", previousBlock, err)
				}
				err = bt.SetLastBlockInDataTable(int64(previousBlock))
				if err != nil {
					return fmt.Errorf("error setting last block [%v] in data table: %w", previousBlock, err)
				}
```